circus-train-housekeeping/src/main/java/com/hotels/bdp/circustrain/housekeeping/CircusTrainHousekeepingConfiguration.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (C) 2016-2017 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.circustrain.housekeeping;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+import com.hotels.bdp.circustrain.api.event.HousekeepingListener;
+import com.hotels.housekeeping.repository.LegacyReplicaPathRepository;
+import com.hotels.housekeeping.service.HousekeepingService;
+import com.hotels.housekeeping.service.impl.FileSystemHousekeepingService;
+
+@Configuration
+@ComponentScan("com.hotels.housekeeping")
```

Comment:
are we all fine with this package name or should this be `com.hotels.bdp.housekeeping` instead?

Reply:
I think it's generally better to avoid putting our team names into package names when going external but with CT itself that would have been a massive backwards incompatible change so we didn't do that there. Here there are probably very few, if any, people using these classes outside of CT so I don't mind changing the package like this.